docs/UserGuide.md

```diff
@@ -214,17 +205,16 @@ Examples:
 * `add n/Product Pitch i/Product pitch for James at MBS from/2024-10-31 13:00 to/2024-10-31 15:30`
 * `add n/Show buyer 3 houses i/Meet with James Jimes to show him 3 houses that fit his expectations from/2024-10-30 09:30 to/2024-10-30 10:30`
 
-### Listing all meet-ups : `list`
+### Viewing all meet-ups : `view`
```

Comment:
Good change in the language used, helps reduce technical jargon for all users to understand easier.